MotorizationRate.ipynb

```diff
@@ -148,7 +148,7 @@
     }
```

Comment:
<p>probably*, surely*</p><p> </p><p>Issue: <code>.value_counts()</code>  means the counts are on the full content of the "cells": if the last category "contains" have entries that <em>contain</em> other entries, you probably need a group by and make clear categories.</p><p>If the indexing column is a primary key (no duplicate values) then no, the last category does NOT contain others.</p><p> </p><p>Note that it is bad practive to designate "last category" when the code does not ensure order of categories in the chat. If rerunning the notebook leads to another category being the last presented, this cancels your explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Adanel/MotorizationRate/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>